e2e2/test/cases/netpol/main_test.go

```diff
@@ -0,0 +1,333 @@
+			},
+		},
+	}
+
+	if err := client.Resources().Create(ctx, deploy); err != nil {
+		return err
+	}
+
+	err := wait.For(conditions.New(client.Resources()).DeploymentConditionMatch(deploy, appsv1.DeploymentAvailable, v1.ConditionTrue),
+		wait.WithTimeout(time.Minute*5))
+	if err != nil {
+		return err
+	}
+
+	service := &v1.Service{
+		ObjectMeta: metav1.ObjectMeta{Name: name, Namespace: namespace},
+		Spec: v1.ServiceSpec{
+			Ports:    []v1.ServicePort{{Name: name, Protocol: "TCP", Port: 80}},
+			Selector: labels,
+		},
+	}
+
+	if err := client.Resources().Create(ctx, service); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func TestNetworkPolicyCases(t *testing.T) {
+
+	manifests := "manifests/network-policy.yaml"
+
+	allowAll := features.New("allowAll").
+		WithLabel("suite", "netpol").
+		WithLabel("policy", "none").
+		Setup(func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			return ctx
+		}).
+		Assess("curl from A to B succeeds", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			client, err := cfg.NewClient()
+			if err != nil {
+				return ctx
+			}
+			pods := &corev1.PodList{}
+			namespace := "a"
+			containerName := "a-server"
+			err = client.Resources("a").List(context.TODO(), pods)
+			if err != nil || pods.Items == nil {
+				t.Error("error while getting pods", err)
+			}
+			podName := pods.Items[0].Name
+
+			var stdout, stderr bytes.Buffer
+			command := []string{"curl", "-m", "2", "-I", "http://b-server.b:80"}
+			client.Resources().ExecInPod(context.TODO(), namespace, podName, containerName, command, &stdout, &stderr)
+
+			httpStatus := strings.Split(stdout.String(), "\n")[0]
+			if !strings.Contains(httpStatus, "200") {
+				t.Fatal("Couldn't connect to server B")
+			}
+			return ctx
+
+		}).
+		Assess("curl from C to A succeeds", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			client, err := cfg.NewClient()
+			if err != nil {
+				return ctx
+			}
+			namespace := "c"
+			containerName := "c-server"
+			pods := &corev1.PodList{}
+			err = client.Resources("c").List(context.TODO(), pods)
+			if err != nil || pods.Items == nil {
+				t.Error("error while getting pods", err)
+			}
+			podName := pods.Items[0].Name
+
+			var stdout, stderr bytes.Buffer
+			command := []string{"curl", "-m", "2", "-I", "http://a-server.a:80"}
+			client.Resources().ExecInPod(context.TODO(), namespace, podName, containerName, command, &stdout, &stderr)
+
+			httpStatus := strings.Split(stdout.String(), "\n")[0]
+			if !strings.Contains(httpStatus, "200") {
+				t.Fatal("Couldn't connect to server A")
+			}
+			return ctx
+		}).
+		Feature()
+
+	blockCToA := features.New("blockCToA").
+		WithLabel("suite", "netpol").
+		Setup(func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			err := fwext.ApplyFiles(cfg.Client().RESTConfig(), manifests)
```

Comment:
Can you swap this out for `ApplyManifests`? We need to make sure the manifests are embedded in the binary, ex: https://github.com/aws/aws-k8s-tester/blob/main/e2e2/test/cases/nvidia/mpi_test.go#L23-L30

Reply:
I've updated to not use the manifest file and define the network policy using go client